### PR TITLE
Upgrade tutorial for 12.1.0.Final

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-12932

### CODE REVIEW

**Comment by karesti** [comment]
@oraNod upgraded

File: README.adoc [hunk 1]
```diff
@@ -95,21 +95,24 @@ You can do one of the following:
 * Pull the container image and run with https://www.docker.com/[Docker] or https://podman.io/[Podman].
 * Download the server distribution and extract it to your filesystem.
 
+[IMPORTANT]
 .Credentials
```
Comment (oraNod) [inline README.adoc:99]:
```suggestion
```

File: README.adoc [hunk 2]
```diff
@@ -95,21 +95,24 @@ You can do one of the following:
...
-By default, Infinispan Server requires user authentication. This tutorial uses
-`admin` and `pass` credentials but you can use any username and password.
+By default, Infinispan Server *requires user authentication* and *authorization* is implicit since
```
Comment (oraNod) [inline README.adoc:100]:
```suggestion
By default, Infinispan Server requires user *authentication* and *authorization*.
```

File: README.adoc [hunk 3]
```diff
@@ -95,21 +95,24 @@ You can do one of the following:
...
+12.1. *Creating a user named `admin` gives you administrative access to Infinispan Server.*
```
Comment (oraNod) [inline README.adoc:101]:
```suggestion
For the purposes of this tutorial, create a user named `admin` so you get administrative access to Infinispan Server.
```

**Review by oraNod — CHANGES_REQUESTED** [review]
Thanks, Katia. Sorry I missed this yesterday. Just a couple minor changes and I think it's good to go then.

**Review by oraNod — APPROVED** [review]
LGTM

**Comment by oraNod** [comment]
Thanks @karesti I wasn't able to squash and merge though so maybe the git log won't be so clean...

**Pull request merged**